tests/test-refresh-monitor.c

```diff
@@ -966,13 +966,36 @@ test_sdi_get_desktop_file_from_snap_two_valid_apps_none_right(void) {
   g_assert_null(app_info);
 }
 
+static void
+test_refresh_inhibit_with_negative_value_dont_shows_notifications(void) {
+  // https://github.com/canonical/snapd-desktop-integration/issues/135
+  reset_mock_snapd();
+  MockSnap *snap1 = mock_snapd_add_snap(snapd, "snap1");
+  set_snap_as_inhibited(snap1,
+                        TIME_TO_SHOW_REMAINING_TIME_BEFORE_FORCED_REFRESH + 10);
+  MockSnap *snap2 = mock_snapd_add_snap(snapd, "snap2");
+  set_snap_as_inhibited(snap2, -400);
+  new_notice("refresh-inhibit");
+  g_assert_true(wait_for_notice());
+
+  g_autoptr(ReceivedSignalData) data2 =
```

Comment:
nit:
not sure why this is called `data2`

Reply:
Ops... because I copied another test and deleted a check before. I'll change that.

Reply:
Done.